cores/arduino/stm32/twi.c

```diff
@@ -354,6 +367,16 @@ static uint32_t i2c_computeTiming(uint32_t clkSrcFreq, uint32_t i2c_speed)
   uint8_t presc, scldel, sdadel;
   uint32_t tafdel_min, tafdel_max;
 
+  if (i2c_speed > I2C_SPEED_FREQ_NUMBER) {
+    return ret;
+  }
+  // Don't compute timing if already available value for the requested spped with the same I2C input frequency
```

Comment:
In C file preferred comment syntax: `/*...*/`
+ small typo fix
```suggestion
  //* Don't compute timing if already available value for the requested speed with the same I2C input frequency */
```